Sources/MintKit/Mint.swift

```diff
@@ -188,9 +199,91 @@ public struct Mint {
         try addPackage(git: packagePath.gitPath, path: packagePath.packagePath)
         print("🌱  Installed \(package.commandVersion)".green)
 
+        if global {
+            try installGlobal(packagePath: packagePath)
+        }
+
+
         try? packageCheckoutPath.delete()
     }
 
+    func installGlobal(packagePath: PackagePath) throws {
+
+        let installPath = installsPath + packagePath.package.name
+        try installsPath.mkpath()
+
+        // copy to global installs
+        try? installPath.delete()
+        let output = main.run(bash: "ln -s \(packagePath.commandPath.absolute().string) \(installPath.absolute().string)")
+        guard output.succeeded else {
+            print("🌱  Could not install \(packagePath.package.commandVersion) globally")
+            return
+        }
+
+        let exportCommand = "export PATH=\"\(installsPath):$PATH\""
+
+        // add $PATH to current shell
+        // FIXME: this doesn't seem to work
+        main.run(bash: exportCommand)
+
+        // add $PATH to common files
+        let exportSection = "# Register Mint install path\n\(exportCommand)"
+
+        let files = [
+            "~/.bash_profile",
+            "~/.bashrc",
+            "~/.zshenv",
+            ]
+
+        try files.forEach {
+            let file = Path($0).absolute()
+            if file.exists {
```

Comment:
I am not sure if the use of a shell guarantees it will have a log‐on script. If the user has not configured anything yet, I do not know if the file would just be empty or completely non‐existent.

It is a clever way of checking which shells the user likely prefers, so I recommend keeping it this way for now unless someone files a bug.